scene/2d/ray_cast_2d.h

```diff
@@ -50,6 +50,7 @@ class RayCast2D : public Node2D {
 
 	bool collide_with_areas = false;
 	bool collide_with_bodies = true;
+	void _draw_debug_shape();
```

Comment:
And just a little detail, but an empty line before the method would be nice for readability.